renderdoc/driver/vulkan/vk_postvs.cpp

```diff
@@ -2879,7 +2879,8 @@ void VulkanReplay::FetchTessGSOut(uint32_t eventId, VulkanRenderState &state)
   }
 
   if(state.dynamicRendering.viewMask > 1 ||
-     !creationInfo.m_RenderPass[state.GetRenderPass()].subpasses[state.subpass].multiviews.empty())
+     (state.GetRenderPass() != ResourceId() &&
+      !creationInfo.m_RenderPass[state.GetRenderPass()].subpasses[state.subpass].multiviews.empty()))
   {
     RDCWARN("Multipass is active for this draw, no GS/Tess mesh output is available");
```

Comment:
Unrelated to the issue itself, but I wonder if this message should said "multiview" instead? @baldurk

Reply:
Technically yes but this is a developer-only message that is not user visible, so it doesn't really matter as the intent is clear.